### PR TITLE
Create topic_model.py

### PR DESCRIPTION
**Pull request by BrandonSmith710**
Description: This file proposes a new method of topic modeling, called Non-Negative Matrix Factorization. The function, "topicizer" will accept a list of meeting notes, clean and vectorize the text, then return the desired number of topics; all from a minimally dependent environment.

Dependencies: sklearn.feature_extraction.text, sklearn.decomposition

Type of change:
- [ ] New feature
- [ ] This change requires a documentation update

Checklist:
- [ ] My code follows PEP8 style guide
- [ ] I have removed unnecessary print statements from my code
- [ ] I have made corresponding changes to the documentation if necessary
- [ ] My changes generate no errors
- [ ] No commented-out code
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by BrandonSmith710** [comment]
https://www.loom.com/share/b272dfd36f6e42e3ab271b67ea043d89

**Review by hollberg — APPROVED** [review]
Nice work! Adding the ability to classify text (such as meeting notes) by topic continues to build on the insights administrators will be able to extract from these artifacts.

File: app/topic_model.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+from sklearn.decomposition import NMF
+from sklearn.feature_extraction.text import TfidfVectorizer
+import re
+
+# will be ignoring terms with a document frequency greater than .8
+tfidf_vect = TfidfVectorizer(max_df=.8, min_df=2, stop_words='english')
+
+def cleaner(text: str):
+  '''
```
Comment (jinjahninjah) [inline app/topic_model.py:9]: PEP8 wants double quotes for docstrings

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
Great first attempt! There are a few things with PEP8 that need fixing like docstrings quotes and indentation should be 4 spaces not 2. You might want to look at your ```topicizer()``` method as it's currently throwing an error. Also, throw the sklearn library in the requirements.txt file 👍

**Comment by BrandonSmith710** [comment]
> Great first attempt! There are a few things with PEP8 that need fixing like docstrings quotes and indentation should be 4 spaces not 2. You might want to look at your `topicizer()` method as it's currently throwing an error. Also, throw the sklearn library in the requirements.txt file 👍
Oops, thanks Royce! Will do.

**Review by OlatomiAdigun — APPROVED** [review]
Nice work. Good explanatory video

**Pull request closed without merging**